.flake8

```
@@ -30,15 +29,15 @@
 extend-ignore = A, D, N8, SIM, RST, TYP, E301, E302, E305, E501, E701
```

According to the principle you articulated in #9715 ;)

```suggestion
extend-ignore = A, B, D, N8, SIM, RST, TYP, E301, E302, E305, E501, E701
```

---

Yes thank you!
Even more so devs will already have it installed in their venv